ipynb/examples/energy_meter/EnergyMeter_AEP.ipynb

```diff
@@ -0,0 +1,413 @@
+     ]
+    }
+   ],
+   "source": [
+    "# Create and RTApp RAMP task\n",
+    "rtapp = RTA(te.target, 'ramp', calibration=te.calibration())\n",
+    "rtapp.conf(kind='profile',\n",
+    "           params={\n",
+    "                'ramp' : Ramp(\n",
+    "                    start_pct =  60,\n",
+    "                    end_pct   =  20,\n",
+    "                    delta_pct =   5,\n",
+    "                    time_s    =   0.5).get()\n",
+    "          })\n",
+    "\n",
+    "# EnergyMeter Start\n",
+    "te.emeter.reset()\n",
+    "\n",
+    "rtapp.run(out_dir=te.res_dir)\n",
+    "\n",
+    "# EnergyMeter Stop and samples collection\n",
+    "er = te.emeter.report(te.res_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:03:27  INFO    : Collected data:\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[01;34m/data/lisa/results/EnergyMeter_AEP\u001b[00m\r\n",
+      "├── energy.json\r\n",
+      "├── output.log\r\n",
+      "├── ramp_00.json\r\n",
+      "├── rt-app-ramp-0.log\r\n",
+      "└── samples.csv\r\n",
+      "\r\n",
+      "0 directories, 5 files\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "logging.info(\"Collected data:\")\n",
+    "!tree $te.res_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Power Measurements Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:03:31  INFO    : Measured channels energy:\n",
+      "02:03:31  INFO    : {'BAT': '0.026167'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "logging.info(\"Measured channels energy:\")\n",
+    "logging.info(\"%s\", er.channels)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:03:35  INFO    : Returned energy file:\n",
+      "02:03:35  INFO    :   /data/lisa/results/EnergyMeter_AEP/energy.json\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\r\n",
+      "    \"BAT\": \"0.026167\"\r\n",
+      "}"
+     ]
+    }
+   ],
+   "source": [
+    "logging.info(\"Returned energy file:\")\n",
+    "logging.info(\"  %s\", er.report_file)\n",
+    "!cat $er.report_file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:06:33  INFO    : BAT stats (head)\n"
```

Comment:
Maybe something a bit more detailed, e.g. "Samples collected for the BAT channel (only first 10)" ?